python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -331,3 +331,19 @@ def get_units(value):
         return _unit_map[value.lower()]
 
     return value
+
+
+class DateOffset(object):
+    def __init__(self, months):
+        self._months = months
```

Comment:
We should follow the signature of Pandas `DateOffset` here: https://pandas.pydata.org/docs/reference/api/pandas.tseries.offsets.DateOffset.html

We should also validate that we support the parameters that are used.